docs/docs/Contributing/contributing-github-issues.md

```diff
@@ -4,21 +4,14 @@ sidebar_position: 2
 slug: /contributing-github-issues
 ---
 
+Our [issues](https://github.com/langflow-ai/langflow/issues) page is kept up to date with bugs, improvements, and feature requests. There is a taxonomy of labels to help with sorting and discovery of issues of interest. For an overview of the system we use to tag our issues and pull requests, see the Langflow repo's [labels page](https://github.com/langflow-ai/langflow/labels).
 
+## GitHub Discussions
 
-Our [issues](https://github.com/langflow-ai/langflow/issues) page is kept up to date with bugs, improvements, and feature requests. There is a taxonomy of labels to help with sorting and discovery of issues of interest.
+If you're looking for help with your code, consider posting a question on the Langflow [GitHub Discussions board](https://github.com/langflow-ai/langflow/discussions). Please understand that we won't be able to provide individual support via email. We also believe that help is much more valuable if it's **shared publicly**, so that more people can benefit from it.
```

Comment:
```suggestion
If you're looking for help with your code, consider posting a question on the Langflow [GitHub Discussions board](https://github.com/langflow-ai/langflow/discussions). The Langflow team cannot provide individual support via email. We also believe that help is much more valuable if it's **shared publicly**, so that more people can benefit from it.
```